src/lib/y2network/connection_config/base.rb

```diff
@@ -31,10 +31,12 @@ class Base
       # no specific interface then it could be activated by the first available
       # device.
       #
-      # #FIXME: Maybe it could be a matcher instead of an Interface, or just a
+      # @return [String] Connection name
+      attr_accessor :name
```

Comment:
so connection name can be different to interface name and in sysconfig case it is same, right?

Reply:
In network manager them can be different. In sysconfig it is basically the ifcfg name.